charts/em-hrs-api/values.preview.template.yaml

```diff
@@ -47,8 +47,18 @@ java:
     CVP_STORAGE_CONNECTION_STRING: "DefaultEndpointsProtocol=https;AccountName=$(CVP_BLOB_ACCOUNT_NAME);AccountKey=$(CVP_BLOB_ACCESS_KEY);EndpointSuffix=core.windows.net"
     VH_STORAGE_CONNECTION_STRING: "DefaultEndpointsProtocol=https;AccountName=$(VH_BLOB_ACCOUNT_NAME);AccountKey=$(VH_BLOB_ACCESS_KEY);EndpointSuffix=core.windows.net"
     STORAGEACCOUNT_PRIMARY_CONNECTION_STRING: "DefaultEndpointsProtocol=https;AccountName=$(HRS_BLOB_ACCOUNT_NAME);AccountKey=$(HRS_BLOB_ACCESS_KEY);EndpointSuffix=core.windows.net"
+    CASE_DOCUMENT_AM_URL: http://ccd-case-document-am-api-aat.service.core-compute-aat.internal
```

Comment:
are you sure this is correct?
we should use this?
https://github.com/hmcts/em-hrs-api/pull/2444/files#diff-f7c2359e2f182fbc6a81ac290900a790ee215eb8f5003b2d4dac022f44461c30R159